packages/components/ng-ovh-onboarding/README.md

```diff
@@ -0,0 +1,39 @@
+# ng-ovh-onboarding
```

Comment:
There is already a PR to expose the `empty` component from PCI to every packages through the `ng-layout-helpers` package. https://github.com/ovh/manager/pull/3846

Reply:
👍